ons-spark/_toc.yml

```diff
@@ -17,6 +17,9 @@ parts:
   - file: spark-functions/union_dataframes_with_different_columns
   - file: spark-functions/padding
   - file: spark-functions/sampling
+- caption: Understanding and Optimising Spark
```

Comment:
I just called this Spark Concepts but I much prefer your version, will change mine.